PoshBot/Public/New-PoshBotCardResponse.ps1

```diff
@@ -127,15 +130,17 @@ function New-PoshBotCardResponse {
                 throw [System.Management.Automation.ValidationMetadataException]$msg
             }
         })]
-        [string]$Color = '#D3D3D3'
+        [string]$Color = '#D3D3D3',
+
+        [object]$CustomData
     )
 
     $response = [ordered]@{
         PSTypeName = 'PoshBot.Card.Response'
         Type = $Type
         Text = $Text.Trim()
         Private = $PSBoundParameters.ContainsKey('Private')
```

Comment:
Is `Private` just here as a reservation or can it be removed as well? The current parameters don't apparently have `Private` included.

Reply:
For sure!